### ISSUE TITLE
Change progress bar on main page

### ISSUE THREAD

**Issue by m4ktub** [acceptbitcoincash/acceptbitcoincash#1260]
The progress bar currently shows 50%, at the bottom. I believe it should be `n_merchants_bch / n_merchants_any` instead of `n_merchants_bch / n_merchants`. Those 50% could 1% if you simply added more merchants so it does not seem a very good statistic. Also, an higher number would look better 😃.

**Comment by chaintip** [comment]
`0.01225964 BCH`| `~   5.90 USD` has [been tipped](https://explorer.bitcoin.com/bch/address/bitcoincash:qqh0ld4233f5k55zk2m3pn3kt99r788zpccmyegtkv) to this issue by @m4ktub.
To claim [this bounty](http://github.chaintip.org/chaintip/bounties/issues/45), get a pull request merged with ```@chaintip fixes #1260``` in the creation comment.

**Comment by EchterAgo** [comment]
I agree with this change. If we don't keep adding merchants accepting no crypto currency then the progress bar is kind of pointless. We should show how many crypto merchants accept Bitcoin Cash.

**Comment by chaintip** [comment]
Thank you @gitcash_testtip! `0.01319247 BCH`| `~   7.09 USD` has now been tipped to this issue.
To claim [this bounty](http://github.chaintip.org/chaintip/bounties/issues/45), get a pull request merged with ```@chaintip fixes #1260``` in the creation comment.

**Comment by chaintip** [comment]
Thank you @gitcash_testtip! `0.0141253 BCH`| `~   7.60 USD` has now been tipped to this issue.
To claim [this bounty](http://github.chaintip.org/chaintip/bounties/issues/45), get a pull request merged with ```@chaintip fixes #1260``` in the creation comment.

**Comment by chaintip** [comment]
Thank you @gitcash_testtip! `0.01505987 BCH`| `~   8.10 USD` has now been tipped to this issue.
To claim [this bounty](http://github.chaintip.org/chaintip/bounties/issues/45), get a pull request merged with ```@chaintip fixes #1260``` in the creation comment.

**Comment by kenman345** [comment]
@t0p0lin0 I am happy to change it but I wanted to get some input from the original team member that added the progress bar. 

I looked at the Semantics UI element we are using and it does not contain any options to have two progress indicators within one progress bar. So we would have to change the current one or add a second one to the page.

**Comment by kenman345** [comment]
we have this progress data also in our stats.json we generate. I just want to get some more opinions on this and maybe how to be clear it has changed to the rest of the people. 

I still feel like the progress bar makes a lot of sense to the page itself and that if we only have it based off the total merchants on the page because one day we hope to be 100% of crypto accepting merchants accept BCH and then the progress bar needs to change back?

**Comment by chaintip** [comment]
`0.00280023 BCH`| `~   1.26 USD` has now been tipped to this issue.
To claim [this bounty](http://github.chaintip.org/chaintip/bounties/issues/45), get a pull request merged with ```@chaintip fixes #1260``` in the creation comment.

**Comment by kenman345** [comment]
I am tempted to close this. The goal of the site is to get bitcoin cash accepted everywhere, not just where places already accept crypto. If anything we can add this type of progress bar to the adoption page, but the main pages should be indicating the value/progress based off the same numbers you see for each section that indicates of how many listings the amount that accept BCH. 

I do not have any desire to confuse people and as I pointed out above, once BCH is accepted everywhere other cryptocurrencies are accepted, then we would have to change the progress bar back otherwise we would have users complain why does our progress bar say 100% yet they only have 3 options out of 20 in X or Y Category.
